### PR TITLE
Alex/open3d ml tutorials

### PR DESCRIPTION
**Pull request by INF800**
### Re-write condensed tutorials based on suggestions.

Changes were made to 3 tutorial notebooks

- docs/tutorial/notebook/01_config_files.ipynb
- docs/tutorial/notebook/02_datasets.ipynb
- docs/tutorial/notebook/03_train_model.ipynb

Suggested changes were:

1. The notebooks must look similar to already existing open3d tutorial notebooks (all notebooks)
2. Run all the notebooks files and check if they work locally (all notebooks)
3. Working directory must be `docs/tutorial/notebook/` for all data, model checkpoint logs etc. (all notebooks)
4. Remove garbage-in garbage-out dataset graphic. (`01_config_files.ipynb`)
5. Use consistent vector notations (`01_config_files.ipynb`)
6. Change to right-handed coordinate system (`02_datasets.ipynb`)
7. Change colour of axes in the coordinate system (`02_datasets.ipynb`)

Some key-issues:

1. Some cells of `02_datasets.ipynb` will not run because the notebook does not have access to **smaller version** of semanticKITTI dataset. @sanskar107 can you share a downloadable link for that dataset that I can `wget` in the the notebook?

**Please note:** I will remove `Inference_on_a_custom_data.ipynb`, `reading_a_config_file.ipynb`, `reading_a_dataset.ipynb`, `train_ss_model_using_pytorch.ipynb` and `train_ss_model_using_tensorflow.ipynb` because they are essentially duplicates.

### CODE REVIEW

File: docs/tutorial/notebook/01_config_files.ipynb [hunk 1]
```diff
@@ -0,0 +1,910 @@
+{
```
Comment (ssheorey) [inline docs/tutorial/notebook/01_config_files.ipynb:1]: <p>You can access configuration values as object attributes (<code>cfg.{property_name})</code> or dictionary key values (<code>cfg['{property_name}']</code>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

File: docs/tutorial/notebook/02_datasets.ipynb [hunk 2]
```diff
@@ -0,0 +1,449 @@
+{
```
Comment (ssheorey) [inline docs/tutorial/notebook/02_datasets.ipynb:1]: <p>Remove Open3D-ML for consistency</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (ssheorey) [inline docs/tutorial/notebook/02_datasets.ipynb:1]: <p>Standard convention for object detection lidar datasets (e.g. KITTI / Waymo) is x(red) = forward, <em>y (green)</em> = left, <em>z</em> (blue) = <em>up</em>. Add XYZ labels to the axis.</p><p> </p><p>Mention that datasets may instead of r (lidar reflectance) as feature, or no features at all.</p><p> </p><p>A dataset is partitioned into train(ing), valid(ation) and test(ing) splits. Models use data from the training split to update network weights and performance is measured on the validation split. Hyperparameters can be adjusted to optimize performance on the validation split. The test split is never used during the training step and is only used to report the final performance.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

File: docs/tutorial/notebook/03_train_model.ipynb [hunk 3]
```diff
@@ -0,0 +1,583 @@
+{
```
Comment (ssheorey) [inline docs/tutorial/notebook/03_train_model.ipynb:1]: <p>Both PyTorch and TF can be installed. User code just needs to choose one inside a specific python file.</p><p><code>requirements-torch.txt </code>if Nvidia GPU is not available. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (ssheorey) [inline docs/tutorial/notebook/03_train_model.ipynb:1]: <p>This section should be in the datasets tutorial.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (ssheorey) [inline docs/tutorial/notebook/03_train_model.ipynb:1]: <p>holdout -&gt; test split for consistency</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Comment (ssheorey) [inline docs/tutorial/notebook/03_train_model.ipynb:1]: <p>Also talk about:</p><p>Continuing training from a checkpoint.</p><p>Point to the tensorboard tutorial for visualizing training progress.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

File: docs/tutorial/notebook/01_config_files.ipynb (hunk 1 above)
Comment (ssheorey) [inline docs/tutorial/notebook/01_config_files.ipynb:1]: <p>See Readme.md for how to import Config</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

File: docs/tutorial/notebook/03_train_model.ipynb (hunk 3 above)
Comment (ssheorey) [inline docs/tutorial/notebook/03_train_model.ipynb:1]: <p>remove this listing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

File: docs/tutorial/notebook/01_config_files.ipynb (hunk 1 above)
Comment (INF800) [reply docs/tutorial/notebook/01_config_files.ipynb:1]: <p>Done</p>
